### DISCUSSION TITLE
v1.20.0 draft

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#582] [Draft patch notes]
**Features:**
Initial native code for verified mod auto-downloading ([Launcher#545](https://github.com/R2Northstar/NorthstarLauncher/pull/545))
Plugins v3 ([Launcher#472](https://github.com/R2Northstar/NorthstarLauncher/pull/472), [Mods#652](https://github.com/R2Northstar/NorthstarMods/pull/652), [#583](https://github.com/R2Northstar/Northstar/pull/583))
Expose `Northstar.dll` version to squirrel ([Launcher#558](https://github.com/R2Northstar/NorthstarLauncher/pull/558))
Convar print commands ([Launcher#539](https://github.com/R2Northstar/NorthstarLauncher/pull/539))
Remove "Launch Multiplayer" button ([Mods#719](https://github.com/R2Northstar/NorthstarMods/pull/719))
Re-implement projectile kill replays ([Mods#723](https://github.com/R2Northstar/NorthstarMods/pull/723))

**Fixes:**
Plugin related log error expecting more args ([Launcher#591](https://github.com/R2Northstar/NorthstarLauncher/pull/591))

**Contributors:**
@ASpoonPlaysGames @Alystrasz @catornot @GeckoEidechse @H0L0theBard

**Comment by graceburger** [comment]
i must say this is pretty poggers
